py-polars/polars/internals/expr.py

```diff
@@ -58,12 +58,7 @@ def _from_pyexpr(pyexpr: "PyExpr") -> "Expr":
         return self
 
     def __to_pyexpr(self, other: Any) -> "PyExpr":
-        if isinstance(other, PyExpr):
-            return other
```

Comment:
Note we never passed in a `PyExpr` into this, so we can defer to `self.__to_expr` and unpack the PyExpr